bridge_types/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "bridge_types"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+derive_more = "0.99.17"
```

Comment:
I don't think you are actually using this dep, if not remove it.  The lower down the workspace stack the less deps we should have (this seems to be below the previous bottom, the vm), I just added the first to the vm workspace (for unicode string support) and it still smarts...  If you do need this dep then it should be fine, I am pretty sure this is a "blessed" crate but if we don't need it don't add it.